src/flutter/shell/platform/linux_embedded/plugin/mouse_cursor_plugin.cc

```diff
@@ -14,7 +14,8 @@
 // /usr/include/X11/X.h:350:21: note: expanded from macro 'Success'
 // #define Success            0    /* everything's okay */
 // ----------------------------------------------------------------
-#if defined(DISPLAY_BACKEND_TYPE_X11)
+#if defined(DISPLAY_BACKEND_TYPE_X11) || \
+    defined(DISPLAY_BACKEND_TYPE_DRM_EGLSTREAM)
```

Comment:
If `gbm.h` was not included, the same error occurred because  X11-related files are included by `eglplatform.h`. Since I could avoid the error by defining `EGL_NO_X11`, I removed it.

Reply:
Will you add that reason in `cmake/build.cmake` file?

Reply:
I added the reason in `cmake/build.cmake` file.